src/describe/UNSTABLE__paginatedResource.ts

```diff
@@ -94,6 +94,25 @@ const UNSTABLE__describePaginatedResource = <IdentityType, ResourceDataType>(
       body,
     });
 
+  Resource.refresh = async (refreshIdentity: IdentityType) => {
+    return UNSTABLE__yieldWholePaginatedResource<
+      IdentityType,
+      ResourceDataType
+    >({
+      // @ts-expect-error TODO: I can't figure out how to let this function accept an incomplete resource
+      // type. The resource type will be complete any time the function actually runs.
+      Resource,
+      body: (identity, startIndex, endIndex, data) => {
+        if (
+          refreshIdentity !== undefined &&
+          !Resource.getAreIdentitiesEqual(refreshIdentity, identity)
+        )
+          return data;
```

Comment:
Maybe I'm misinterpreting  -- but I thought if the identities are not equal, we're suppose to do a `get`, not return the existing data 🤔

Reply:
Since the identity is going to be search parameters in most cases, this translates to:

Go through all of the data in cache for this resource. If the data is associated with another search, move on (or, return what we have in the cache already). Otherwise, re-fetch the data.

Reply:
Hmm -- maybe I'm not understanding still, does this mean the intended usage should look like this?
```
paginatedResource.refresh(); // does a re-fetch for data
paginatedResource.refresh({ ...same search params }); // does a re-fetch for data
paginatedResource.refresh({ ...anything else }); // returns cached data
```

Reply:
I think it should just be either `paginatedResource.refresh()` or `paginatedResource.refresh(identity)`.

```
paginatedResource.refresh(); // refetch (call Resource.get) on all Resource usage
paginatedResource.refresh({ ...same search params }); // only re-fetch on Resource.use that matches the identity
```